File: milestone 2/python_modules/module00/ex4/ft_water_reminder.cpp
#include <iostream>

void	ft_water_reminder(void)
{
	int	days_since_watering;

	std::cout << "Days since last watering: ";
	std::cin >> days_since_watering;
	if (days_since_watering > 2)
		std::cout << "Water the plants!\n";
	else
		std::cout << "Plants are fine.\n";
}

int	main(void)
{
	ft_water_reminder();
	return 0;
}

File: milestone 2/python_modules/module00/ex4/ft_water_reminder.java
import java.util.Scanner;

public class ft_water_reminder {
    public static void main(String[] args)
    {
        System.out.print("Days since last watering: ");
        Scanner scan = new Scanner(System.in);
        int days_since_watering = scan.nextInt();
        if (days_since_watering > 2)
            System.out.println("Water the plants!");
        else
            System.out.println("Plants are fine.");
    }
}